conftest.py

```diff
@@ -108,3 +108,16 @@ def _bias_params(evoked, noise_cov, fwd):
     if not mne.forward.is_fixed_orient(fwd):
         want //= 3
     return evoked, fwd, noise_cov, data_cov, want
+
+
+@pytest.fixture(scope="module",
+                params=["mayavi"])
+def backend_name(request):
+    yield request.param
+
+
+@pytest.yield_fixture
+def backends_3d(backend_name):
+    from mne.viz import use_test_3d_backend
+    with use_test_3d_backend(backend_name):
```

Comment:
Somewhere there should be an import check and a `pytest.skip(...)` call or the like, right? That way we can get rid of the `requires_mayavi` decorators for any tests that use this fixture